src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/ILike.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\ORM\Query\AST\Functions;
+
+/**
+ * Implementation of PostgreSql ILIKE()
+ * @see    https://www.postgresql.org/docs/9.3/functions-matching.html
+ * @since  0.4
+ * @author Martin Georgiev <martin.georgiev@gmail.com>
+ */
+class ILike extends BaseFunction
```

Comment:
```suggestion
class Ilike extends BaseFunction
```
I adopted a certain style for PHP class names that represent PostgreSQL functions. It follows a camel-case variation of the usually snake-case named functions. The idea is that `to_tsvector` function will be represented by a `ToTsvector` class. In the case of `tsmatch` the PHP class would be `Tsmatch`. If possible, I'd like to keep the same style for any newly adopted functions. Therefore could you rename this class to `Ilike`? Note that the file name will also have to change.